src/domain/relay/exception-filters/invalid-transfer.exception-filter.ts

```diff
@@ -0,0 +1,21 @@
+import { Response } from 'express';
+import {
+  Catch,
+  ExceptionFilter,
+  ArgumentsHost,
+  HttpStatus,
+} from '@nestjs/common';
+import { InvalidTransferError } from '@/domain/relay/errors/invalid-transfer.error';
+
+@Catch(InvalidTransferError)
+export class InvalidTransferExceptionFilter implements ExceptionFilter {
+  catch(_: InvalidTransferError, host: ArgumentsHost): void {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+
+    response.status(HttpStatus.UNPROCESSABLE_ENTITY).json({
+      message: 'Invalid transfer.',
```

Comment:
We can use the error message here as it seems to provide even more context. Same as with the others.

Reply:
Changed in 4a9d3787f8f6f122c9eb31a6e5e4cb00acc1c6e4.